src/main/java/io/github/pylonmc/pylon/base/items/BeheadingSword.java

```diff
@@ -0,0 +1,108 @@
+package io.github.pylonmc.pylon.base.items;
+
+import io.github.pylonmc.pylon.base.PylonBase;
+import io.github.pylonmc.pylon.base.PylonItems;
+import io.github.pylonmc.pylon.core.item.PylonItem;
+import io.github.pylonmc.pylon.core.item.PylonItemSchema;
+import io.github.pylonmc.pylon.core.item.base.Weapon;
+import io.github.pylonmc.pylon.core.item.builder.ItemStackBuilder;
+import io.github.pylonmc.pylon.core.recipe.RecipeTypes;
+import io.papermc.paper.datacomponent.DataComponentTypes;
+import io.papermc.paper.datacomponent.item.ItemEnchantments;
+import io.papermc.paper.tag.EntityTags;
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Player;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.event.entity.EntityDeathEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.RecipeChoice;
+import org.bukkit.inventory.ShapedRecipe;
+import org.bukkit.inventory.meta.SkullMeta;
+import org.bukkit.inventory.recipe.CraftingBookCategory;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Map;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.function.Function;
+
+@SuppressWarnings("UnstableApiUsage")
+public class BeheadingSword extends PylonItemSchema {
+    private static final ItemStack SILK_TOUCH_BOOK = ItemStackBuilder.defaultBuilder(Material.ENCHANTED_BOOK, new NamespacedKey(PylonBase.getInstance(), "silk_touch_book_beheading_sword"))
```

Comment:
This will never get translated if it's not registered as a Pylon item, so just do this
```suggestion
    private static final ItemStack SILK_TOUCH_BOOK = ItemStackBuilder.of(Material.ENCHANTED_BOOK)
```